### ISSUE TITLE
A11Y_Oracle Data Migration extension for Azure Data Studio_data migration for oracle_data migration status_Screen Reader: Screen Reader is not announcing "Data Migration status" Heading.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/azuredatastudio#24927]
### 
GitHub Tags:
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopApp; #Benchmark; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #Narrator; #WCAG1.3.1; #Info and Relationships;#STP; #FTP; #ScreenReader; #Win11; #Win32; #NVDA;#Element:Heading;

### Environment Details:
Application Name: Oracle Data Migration extension for Azure Data Studio
Download application: Azure Data Studio.
Version:1.45.1(system setup)
Operating System: Window 11 version 22H2 OS build 25267.1000.
### Repro Steps:
1. Open Azure Data Studio application.
2. Install "Data Migration for Oracle"
3. connect to source connection.
4. Tab till new Migration and invoke on it.
5. After completed 6steps and invoke the "Start Migration" in step-6.
6. Tab till "Data Migration Status" card.
7. Now enable narrator.
8. Tab till refresh button and press enter.
9. Now verify that Narrator is not announcing "Data Migration status" Heading.

### Actual Result:
Screen Reader is not announcing "Data Migration status" Heading.
### Expected Result:
Screen Reader is not announcing "Data Migration status" Heading.
### User Impact:
Screen reader will be unable to know the "Data Migration status " heading. If narrator is not announcing.
### Attachment

https://github.com/microsoft/azuredatastudio/assets/142479541/8e08c250-9737-40f7-af46-8a1e5cc5b675

**Comment by PadminiGariki** [comment]
Primary bug id: https://msdata.visualstudio.com/Migration%20Assistant/_workitems/edit/2661072

**Comment by Swati1700** [comment]
@kisantia Could you please provide the ETA of fix. When the fixes will be available in the environment.
Regressed:02-12-24

**Comment by kisantia** [comment]
@SrivenNookala @Bhavikshah123 please help provide an ETA. Thanks!

**Comment by kisantia** [comment]
Closing since primary bug in ADO was verified and closed
